src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -488,14 +449,7 @@ class Assembler : public AbstractAssembler {
 
 #undef INSN
 
-#define INSN_ENTRY_RELOC(result_type, header)                               \
-  result_type header {                                                      \
-    guarantee(rtype == relocInfo::internal_word_type,                       \
-              "only internal_word_type relocs make sense here");            \
-    relocate(InternalAddress(dest).rspec());                                \
-    IncompressibleRegion ir(this);  /* relocations */
```

Comment:
Here we have an `IncompressibleRegion` thing to keep instructions in `INSN_ENTRY_RELOC` from being compressed by the C extension, for relocations are here. But after moving to different locations in this patch, the `IncompressibleRegion` is missing. Please see the below comments.

Reply:
Thanks for the reivew, fixed.